lldb/include/lldb/API/SBAddressRange.h

```diff
@@ -58,6 +58,8 @@ class LLDB_API SBAddressRange {
   friend class SBFunction;
   friend class SBProcess;
 
+  lldb_private::AddressRange &ref() const;
+
```

Comment:
This is no longer needed right? This PR contains this fix right? https://github.com/llvm/llvm-project/pull/95997